### PR TITLE
Adding Web Search C# cognitive SDK

### PR DESCRIPTION
**Pull request by sudhansu88**
This also updates Entity Search SDK and related namespaces

PR for Swagger Spec: https://github.com/Azure/azure-rest-api-specs/pull/1844

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by sudhansu88** [comment]
Ping ! Can somebody please start looking into this ?

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/SDKs/CognitiveServices/dataPlane/Search/Search/Customizations/ApiKeyServiceClientCredentials.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.CognitiveServices.Search.EntitySearch
     public class ApiKeyServiceClientCredentials : ServiceClientCredentials
     {
         private readonly string subscriptionKey;
-        private const string SDKIdentifier = "EntitySearch.CSharp.1.0";
+        private const string SDKIdentifier = "Search.CSharp.1.0";
```
[the remark this answers is not in the document]
Comment (sudhansu88) [reply src/SDKs/CognitiveServices/dataPlane/Search/Search/Customizations/ApiKeyServiceClientCredentials.cs]: Removed

File: src/SDKs/CognitiveServices/dataPlane/Search/Search/Microsoft.Azure.CognitiveServices.Search.csproj [hunk 2]
```diff
@@ -5,8 +5,8 @@
     <Description>This client library provides access to the Microsoft Cognitive Services Search APIs.</Description>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
```
Comment (markcowl) [inline src/SDKs/CognitiveServices/dataPlane/Search/Search/Microsoft.Azure.CognitiveServices.Search.csproj]: If you want to release this, you should update the package version

**Review by markcowl — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please update version changes in AssemblyInfo.cs as well.

**Comment by sudhansu88** [comment]
Updated Package versions based on feedback. Please let me know further

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Why is this 2.0.0? Upgrading a major version usually means you either have breaking changes or have lot of new functionality. As you are still in preview, what difference does it make if you are 1.1.0-preview. Be conservative with your version numbers for preview.
If you still want to do it have the release notes reflect what are the changes that are going in that makes a compelling reason for it to have 2.0.0 version.

**Comment by sudhansu88** [comment]
Current Change adds Web Search functionality to already existing SDK  [with Entity Search] So I think this would be a major change. But as we are still in preview [ Considering we have not published any SDK versions until now ] we can be conservative here to go with  1.1.0-preview. Will update to  1.1.0-preview.
Thanks !

**Pull request merged**